doc/source/getting_started/comparison/comparison_with_excel.rst

```diff
@@ -0,0 +1,213 @@
+
+``Series``
+~~~~~~~~~~
+
+A ``Series`` is the data structure that represents one column of a ``DataFrame``. Working with a
+``Series`` is analogous to referencing a column of a spreadsheet.
+
+``Index``
+~~~~~~~~~
+
+Every ``DataFrame`` and ``Series`` has an ``Index``, which are labels on the *rows* of the data. In
+pandas, if no index is specified, an integer index is used by default (first row = 0, second row =
+1, and so on), analogous to row headings/numbers in Excel.
+
+In pandas, indexes can be set to one (or multiple) unique values, which is like having a column that
+use use as the row identifier in a worksheet. Unlike Excel, these ``Index`` values can actually be
+used to reference the rows. For example, in Excel, you would reference the first row as ``A1:Z1``,
+while in pandas you could use ``populations.loc['Chicago']``.
+
+Index values are also persistent, so if you re-order the rows in a ``DataFrame``, the label for a
+particular row don't change.
+
+See the :ref:`indexing documentation<indexing>` for much more on how to use an ``Index``
+effectively.
+
+Commonly used Excel functionalities
+-----------------------------------
+
+Fill Handle
+~~~~~~~~~~~
+
+Create a series of numbers following a set pattern in a certain set of cells. In
+Excel this would be done by shift+drag after entering the first number or by
+entering the first two or three values and then dragging.
+
+This can be achieved by creating a series and assigning it to the desired cells.
+
+.. ipython:: python
+
+    df = pd.DataFrame({"AAA": [1] * 8, "BBB": list(range(0, 8))})
+    df
+
+    series = list(range(1, 5))
+    series
+
+    df.loc[2:5, "AAA"] = series
+
+    df
+
+Filters
+~~~~~~~
+
+Filters can be achieved by using slicing.
+
+The examples filter by 0 on column AAA, and also show how to filter by multiple
+values.
+
+.. ipython:: python
+
+   df[df.AAA == 0]
+
+   df[(df.AAA == 0) | (df.AAA == 2)]
+
+
+Drop Duplicates
+~~~~~~~~~~~~~~~
+
+Another commonly used function is Drop Duplicates. This is directly supported in
+pandas.
+
+.. ipython:: python
+
+    df = pd.DataFrame(
+        {
+            "class": ["A", "A", "A", "B", "C", "D"],
+            "student_count": [42, 35, 42, 50, 47, 45],
+            "all_pass": ["Yes", "Yes", "Yes", "No", "No", "Yes"],
+        }
+    )
+
+    df.drop_duplicates()
+
+    df.drop_duplicates(["class", "student_count"])
+
+
+Pivot Table
+~~~~~~~~~~~
+
+This can be achieved by using ``pandas.pivot_table`` for examples and reference,
+please see `pandas.pivot_table <http://pandas.pydata.org/pandas-docs/stable/generated/pandas.pivot_table.html>`__
+
+
+Formulae
+~~~~~~~~
+
+Let's create a new column "girls_count" and try to compute the number of boys in
+each class.
+
+.. ipython:: python
+
+    df["girls_count"] = [21, 12, 21, 31, 23, 17]
+    df
+
+
+    def get_count(row):
+        return row["student_count"] - row["girls_count"]
+
+
+    df["boys_count"] = df.apply(get_count, axis=1)
```

Comment:
better to just show this a direct operation (or offer the .apply as an alternative, not the primary), e.g.

```
df['boys_count'] = df['student_count'] - df['girls_counts']
```

Reply:
Done.